File: TestDatabaseGenerator/Models/PartsMaster.cs
using System;
using System.Collections.Generic;

namespace TestDatabaseGenerator.Models;

public partial class PartsMaster
{
    public long PkTransactId { get; set; }

    public string? PartNumber { get; set; }

    public string? Rwitmx { get; set; }

    public string? Rwwhse { get; set; }

    public string? PartDesignator { get; set; }

    public short? QtyOutForRepr { get; set; }

    public short? Allotment { get; set; }

    public short? InSuspense { get; set; }

    public short? OrderQuantity { get; set; }

    public short? RefnmbrLinksFiles { get; set; }

    public int? NumberOfSpares { get; set; }

    public string? VendorNumber { get; set; }

    public string? VendorNumber1 { get; set; }

    public string? VendorNumber2 { get; set; }

    public short? CurrentPoNmbr { get; set; }

    public string? CurrentPoType { get; set; }

    public short? LastPoNmbr { get; set; }

    public string? LastPoType { get; set; }

    public double? LastPrice { get; set; }

    public double? CurrentPrice { get; set; }

    public double? AveragePrice { get; set; }

    public string? PrcBreakAvail { get; set; }

    public string? NotesOnPart { get; set; }

    public double? AvgVendLeadTm { get; set; }

    public double? AvgReprLeadTm { get; set; }

    public DateTime? LastInventDate { get; set; }

    public short? PhyCycleNmbr { get; set; }

    public DateTime? LastIssueDate { get; set; }

    public DateTime? LastRecvdDate { get; set; }

    public double? GLAcctNbr { get; set; }

    public string? UsedField { get; set; }

    public string? ShelfLocation { get; set; }

    public string? BinLocation { get; set; }

    public short? QtyOnHand { get; set; }

    public short? QuantityOnOrdr { get; set; }

    public short? OrderLevel { get; set; }

    public int? QtyRequested { get; set; }

    public string? ExtraDta { get; set; }

    public string? PhyInvActive { get; set; }

    public double? ThreeMosStaticUse { get; set; }

    public double? SixMosStaticUse { get; set; }

    public double? TwelveMoStaticUse { get; set; }

    public DateTime? DateUpdated { get; set; }

    public string? IssTypeInUse { get; set; }
}

File: SpectrumWeb/wwwroot/Local/parts_master.js
class PartsMasterClass {
    PartNumber;
    Rwitmx;
    Rwwhse;
    PartDesignator;
    QtyOutForRepr;
    Allotment;
    InSuspense;
    OrderQuantity;
    RefnmbrLinksFiles;
    NumberOfSpares;
    Vendor1;
    Vendor2;
    Vendor3;
    CurrentPoNmbr;
    CurrentPoType;
    LastPoNmbr;
    LastPoType;
    LastPrice;
    CurrentPrice;
    AveragePrice;
    PrcBreakAvail;
    NotesOnPart;
    AvgVendLeadTm;
    AvgReprLeadTm;
    LastInventDate;
    PhyCycleNmbr;
    LastIssueDate;
    LastRecvdDate;
    GLAcctNbr;
    Used;
    ShelfLocation;
    BinLocation;
    QtyOnHand;
    QuantityOnOrdr;
    OrderLevel;
    QtyRequested;
    ExtraDta;
    PhyInvActive;
    ThreeMosStaticUse;
    SixMosStaticUse;
    TwelveMoStaticUse;
    DateUpdated;
    IssTypeInUse;
    constructor(
        partNumber,
        rwitmx,
        rwwhse,
        partDesignator,
        qtyOutForRepr,
        allotment,
        inSuspense,
        orderQuantity,
        refnmbrLinksFiles,
        numberOfSpares,
        vendor1,
        vendor2,
        vendor3,
        currentPoNmbr,
        currentPoType,
        lastPoNmbr,
        lastPoType,
        lastPrice,
        currentPrice,
        averagePrice,
        prcBreakAvail,
        notesOnPart,
        avgVendLeadTm,
        avgReprLeadTm,
        lastInventDate,
        phyCycleNmbr,
        lastIssueDate,
        lastRecvdDate,
        gLAcctNbr,
        used,
        shelfLocation,
        binLocation,
        qtyOnHand,
        quantityOnOrdr,
        orderLevel,
        extraDta,
        phyInvActive,
        threeMosStaticUse,
        sixMosStaticUse,
        twelveMoStaticUse,
        dateUpdated,
        issTypeInUse
    ) {
        this.PartNumber = partNumber;
        this.Rwitmx = rwitmx;
        this.Rwwhse = rwwhse;
        this.PartDesignator = partDesignator;
        this.QtyOutForRepr = qtyOutForRepr;
        this.Allotment = allotment;
        this.InSuspense = inSuspense;
        this.OrderQuantity = orderQuantity;
        this.RefnmbrLinksFiles = refnmbrLinksFiles;
        this.NumberOfSpares = numberOfSpares;
        this.Vendor1 = vendor1;
        this.Vendor2 = vendor2;
        this.Vendor3 = vendor3;
        this.CurrentPoNmbr = currentPoNmbr;
        this.CurrentPoType = currentPoType;
        this.LastPoNmbr = lastPoNmbr;
        this.LastPoType = lastPoType;
        this.LastPrice = lastPrice;
        this.CurrentPrice = currentPrice;
        this.AveragePrice = averagePrice;
        this.PrcBreakAvail = prcBreakAvail;
        this.NotesOnPart = notesOnPart;
        this.AvgVendLeadTm = avgVendLeadTm;
        this.AvgReprLeadTm = avgReprLeadTm;
        this.LastInventDate = lastInventDate;
        this.PhyCycleNmbr = phyCycleNmbr;
        this.LastIssueDate = lastIssueDate;
        this.LastRecvdDate = lastRecvdDate;
        this.GLAcctNbr = gLAcctNbr;
        this.Used = used;
        this.ShelfLocation = shelfLocation;
        this.BinLocation = binLocation;
        this.QtyOnHand = qtyOnHand;
        this.QuantityOnOrdr = quantityOnOrdr;
        this.OrderLevel = orderLevel;
        this.QtyRequested = orderLevel;
        this.ExtraDta = extraDta;
        this.PhyInvActive = phyInvActive;
        this.ThreeMosStaticUse = threeMosStaticUse;
        this.SixMosStaticUse = sixMosStaticUse;
        this.TwelveMoStaticUse = twelveMoStaticUse;
        this.DateUpdated = dateUpdated;
        this.IssTypeInUse = issTypeInUse;
    }
}